### PR TITLE
Support setting sentinel password in sentinel mode

### PR DESCRIPTION
**Pull request by Lance726**
This PR is mainly support setting sentinel password in sentinel mode. And by the way, modify the local startup example and add sentinel password and master password.

### CODE REVIEW

**Comment by Lance726** [comment]
@git-hulk When you are free, please help to take a look. Thx ~

**Comment by git-hulk** [comment]
@Lance726 Sure

File: config/docker-image-conf.toml [hunk 1]
```diff
@@ -35,3 +35,4 @@ Addr = "redis:6379"
 # Addr = "localhost:16379,localhost:6380,localhost:6381"
 # MasterName = "mymaster"
 # Password = foobared
+# SentinelPassword = "foobared1"
```
Comment (git-hulk) [inline config/docker-image-conf.toml:38]:
```suggestion
# SentinelPassword = "foobared1"

```

File: scripts/setup.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ -n $(docker ps -q -f "name=lmstfy-test") ]];then
+	cd scripts/redis && docker-compose -p lmstfy-test down -v --remove-orphans && cd ../..
+fi
+
+cd scripts/redis && docker-compose -p lmstfy-test up -d  --remove-orphans && cd ../..
```
Comment (git-hulk) [inline scripts/setup.sh:7]:
```suggestion
cd scripts/redis && docker-compose -p lmstfy-test up -d  --remove-orphans && cd ../..
```

**Review by git-hulk — APPROVED** [review]

**Pull request merged**